### ISSUE TITLE
SSL proxy additional ports

### ISSUE THREAD

**Issue by houshuang** [zodern/meteor-up#706]
Is there a way (using meteor.ssl or the reverse proxy) to add additional ports that should be proxied? We have currently integrated ShareDB into our app, and this requires a separate websocket connection (I've been looking into ways of "multiplexing it" with the Meteor DDP, but not found an easy one). Since our app loads through https, it requires all websocket connections to be SSL as well.

**Comment by zodern** [comment]
I am thinking of adding a new property to the config: `proxy.additionalPorts` which is an array of additional ports to proxy. In the initial implementation it might only allow a certain port, such as `8000`. Would that be work for you?

**Comment by houshuang** [comment]
Yes. We currently do the Nginx setup ourselves, also because we often deploy multiple servers and need load-balancing. Although being able to set up a single production server with https without any other config would be great. We currently need port 3002, and it would be best if we could open this port, however we could also configure our system to use another arbitrary port - has to be different from the core Meteor port though (and in our case, nginx has to support websocket upgrading for that port too).

**Comment by martinsteinorth** [comment]
+1 for this improvement as I am currently dealing with the same requirement. Is there already some progress?

**Comment by martinsteinorth** [comment]
Just found this: https://github.com/zodern/meteor-up/issues/460#issuecomment-283455401
That works as well.
